buildScripts/pipeline-cli.groovy

```diff
@@ -18,17 +18,20 @@
 def VERSION
 def CLI_REPO = "https://github.com/wso2-extensions/identity-tools-cli.git"
 def BRANCH = "master"
-def repo = "wso2-extensions/identity-tools-cli"
```

Comment:
Updating the groovy script added to the repo (as reference), with the content available in the groovy script inside the jenkins job configuration